### PR TITLE
Added Helm charts for Hardware Classification

### PR DESCRIPTION
**Pull request by sharajat**
This PR is regarding Helm charts for Hardware-classification-controller. 
https://github.com/metal3-io/hardware-classification-controller

### CODE REVIEW

File: charts/hardware-classification/values.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+namespace: hcc-system
```
Comment (digambar15) [inline charts/hardware-classification/values.yaml]: hardware-classification

File: values.yaml [hunk 2]
```diff
@@ -10,3 +10,6 @@ kubeadm:
   namespace: default
 baremetal-operator:
   namespace: default
+hardware-classification:
+  namespace: hcc-system
```
Comment (digambar15) [inline values.yaml]: Ditto!

**Review by digambar15 — CHANGES_REQUESTED** [review]

File: charts/hardware-classification/values.yaml (hunk 1 above)
Comment (sharajat) [reply charts/hardware-classification/values.yaml]: Changed namespace.

File: values.yaml (hunk 2 above)
Comment (sharajat) [reply values.yaml]: changed namespace.

**Comment by digambar15** [comment]
looks good

**Review by digambar15 — APPROVED** [review]
Looks Good

**Comment by maelk** [comment]
From https://github.com/metal3-io/metal3-helm-chart/blob/master/OWNERS , it looks like only two people have the approval rights for the review.

**Comment by maelk** [comment]
aside of hard-coding the limits, this looks ok for me.

**Review by ogelbukh — APPROVED** [review]

**Pull request merged**